cirq-core/cirq/ops/controlled_operation.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import itertools
```

Comment:
nit: add newline after module docstring, here and elsewhere.

Reply:
Done